modules/openxr/openxr_api.cpp

```diff
@@ -62,14 +62,46 @@ bool OpenXRAPI::openxr_is_enabled(bool p_check_run_in_editor) {
 		// Disabled for now, using XR inside of the editor we'll be working on during the coming months.
 		return false;
 
-		// bool enabled = GLOBAL_GET("xr/openxr/in_editor"); // EDITOR_GET("xr/openxr/in_editor");
-		// return enabled;
+		/*
+		bool enabled = false;
+		switch (XRServer::get_xr_mode()) {
+			case XRServer::XRMODE_DEFAULT: {
```

Comment:
Can remove the dead code.

Reply:
That's not dead code, that is future code :) Once I start enabling the editor functionality that code will become relevant.

Reply:
Actually, indeed makes sense to just leave this out, I've got a copy to put back in once the editor is working :)